evm_arithmetization/src/cpu/kernel/tests/account_code.rs

```diff
@@ -178,7 +178,10 @@ fn test_extcodesize() -> Result<()> {
         HashMap::from([(keccak(&code), code.clone())]);
     interpreter.run()?;
 
-    assert_eq!(interpreter.stack(), vec![code.len().into()]);
+    assert_eq!(
+        interpreter.stack(),
+        vec![U256::one() << CONTEXT_SCALING_FACTOR, code.len().into()]
+    );
```

Comment:
DQ: why do we expect this to be the content of the stack here? And why only for `EXTCODESIZE` but not for other tests?

Reply:
That's because `extcodesize` (not `sys_extcodesize`) now returns `code_size, codesize_ctx`. I added the returned stack in a comment in the ASM.